uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -194,20 +194,18 @@ private function createDefaultRelationshipTypes() {
    */
   private function makeAllCurrenciesAvailable() {
     $dao = CRM_Core_DAO::executeQuery('SELECT * from civicrm_currency');
+    $result = civicrm_api3('OptionValue', 'get', array(
+      'return' => array("name"),
+      'option_group_id' => "currencies_enabled",
+    ));
+    $enabledCurrency = array_column($result['values'],'name');
```

Comment:
I think `enabledCurrencies` might be a better name since there will most likely be more than one.